src/main/scala/org/apache/spark/streamdm/streams/FileReader.scala

```diff
@@ -97,35 +101,34 @@ class FileReader extends StreamReader with Logging {
   }
 
   /**
-   * Obtains the specification of the examples in the stream.
-   *
-   * @return an ExampleSpecification of the features
-   */
+    * Obtains the specification of the examples in the stream.
+    *
+    * @return an ExampleSpecification of the features
+    */
   override def getExampleSpecification(): ExampleSpecification = {
     init()
     spec
   }
 
   /**
-   * Get one Exmaple from file
-   *
-   * @return an Exmaple
-   */
+    * Get one Example from file
+    *
+    * @return an Example
+    */
   def getExampleFromFile(): Example = {
     var exp: Example = null

```

Comment:
In this case you could use a lower case variable name, e.g., `example` instead of `exp`, still that is not a major issue.